examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/two-connections/README.md

```diff
@@ -0,0 +1,49 @@
+# ECX Fabric Layer2 Single Connection from fabric cloud router to Azure
```

Comment:
This README.md needs to be updated. I missed it when adding the two-connections example. I've submitted a PR on your fork and branch for this to be updated: https://github.com/manu-equinix/terraform-provider-equinix/pull/13

Other than that everything looks good to me 👍

Reply:
Merged